src/olympia/api/tests/test_fields.py

```diff
@@ -0,0 +1,264 @@
+# -*- coding: utf-8 -*-
+from django.core.exceptions import ValidationError
+
+from mock import Mock
+from nose.tools import eq_
+from rest_framework.request import Request
+from rest_framework.serializers import Serializer
+from rest_framework.test import APIRequestFactory
+
+from olympia.addons.models import Addon
+from olympia.api.fields import (ESTranslationSerializerField,
+                                TranslationSerializerField)
+from olympia.amo.tests import TestCase
+from olympia.translations.models import Translation
+
+
+class _TestTranslationSerializerField(object):
```

Comment:
Why does this have a underscore, doesn't this exclude it from running tests like `test_from_native`?